src/app/containers/Article/index.test.jsx

```diff
@@ -44,6 +45,13 @@ describe('ArticleContainer', () => {
         callGetInitialProps();
         expect(fetch.mock.calls[0][0]).toEqual(`/data/${defaultIdParam}.json`);
       });
+
+      it('should validate the id parameter before fetching article data', () => {
```

Comment:
This doesn't test what it should test. It tests that it validates the route (which is doesn't need to as it's in the unit test below) it should test that when the request is invalid that it does not fetch the article data.

Reply:
Changing this will also remove the code climate duplication warning.